package.json

```diff
@@ -3,7 +3,7 @@
     "description" : "Flexible router which can be used in multiple environments",
     "keywords" : ["routes", "event", "observer", "routing", "router"],
     "homepage" : "http://millermedeiros.github.com/crossroads.js/",
-    "version" : "0.12.0",
+    "version" : "0.13.0",
```

Comment:
version bump should be handled during the package release (we run `npm version` before each deploy)